### DISCUSSION TITLE
Add Keyboard APIs

### DISCUSSION THREAD

**Discussion by PureWeen** [CommunityToolkit/Maui#969] [New Feature Discussions]
.NET MAUI is in bad need of some Keyboard APIs

I've created a basic set here.
https://github.com/PureWeen/ShanedlerSamples/tree/main/ShanedlerSamples/Library/Keyboard

I'm thinking we can put this inside MCT for now and once/if these APIs make it into MAUi.Core, we can deprecate the MCT one's in favor of the MAUI ones. 

But for the NET7 timeline these would be VERY useful to have inside MCT.

**Comment by pictos** [comment]
I approve this feature, you can write a proposal (:
